airbyte-cdk/python/docs/concepts/incremental-stream.md

```diff
@@ -0,0 +1,44 @@
+# The Incremental Stream
```

Comment:
This contains HTTP-stream specific information. We should make this purely in terms of incremental, and move HTTP-specific things to the HTTP section.